src/ComplexityVisitor.js

```diff
@@ -1,8 +1,46 @@
 import { GraphQLList, GraphQLNonNull, GraphQLObjectType } from 'graphql';
+import { print } from 'graphql/language/printer';
 import * as IntrospectionTypes from 'graphql/type/introspection';
 
 import CostCalculator from './CostCalculator';
 
+function isSameArguments(arguments1, arguments2) {
+  if (arguments1.length !== arguments2.length) {
+    return false;
+  }
+
+  return arguments1.every((argument1) => {
+    const argument2 = arguments2.find(
+      ({ name }) => name.value === argument1.name.value,
+    );
+
+    if (!argument2) {
+      return false;
+    }
+
+    return print(argument1.value) === print(argument2.value);
+  });
+}
```

Comment:
taken from upstream

Reply:
would suggest leaving some comments re:code that's taken from elsewhere

Reply:
couldn't you do a deepEquals instead? if this is from upstream tho I guess that's ok